Dockerfile

```diff
@@ -1,66 +1,60 @@
-############################################################
-## Dockerfile to build the ga4gh server on mod_wsgi-express
-## Configurable to use a local dataset
-############################################################
-FROM ubuntu
+# Using multi stage to prevent keeping a second copy of the package in the image
+FROM centos:7.6.1810
+RUN yum -y install epel-release
+RUN yum -y install python36-pip.noarch \
+ git \
+ libffi-devel.x86_64 gcc-c++.x86_64 \
+ python36-devel.x86_64 openssl-devel \
+ libxml2-devel.x86_64 libxslt-devel.x86_64  libcurl-devel.x86_64 make gcc  \
+ && pip3 install --upgrade pip setuptools
 
-# Originally created by Steve Hershman GitHub @hershman
-# previously maintained by Alastair Firth, and Maciek Smuga-Otto of the
-# UCSC Genomics Institute
-MAINTAINER David Steinberg <david@resium.com>
+ENV SCHEMA_V=v1.0.0 INGEST_V=v1.3.0
 
-# Update the sources list
-RUN apt-get update  --fix-missing
-RUN apt-get upgrade --yes
+RUN  pip install \
+  git+https://github.com/CanDIG/candig-schemas.git@${SHEMA_V}#egg=candig_schemas  \
+  git+https://github.com/CanDIG/candig-ingest.git@${INGEST_V}#egg=candig_ingest \
+  gevent
 
-# Install packages
-RUN apt-get install -y tar git curl libcurl4-openssl-dev wget dialog \
-    net-tools build-essential python python-dev python-distribute \
-    python-pip zlib1g-dev apache2 libapache2-mod-wsgi libxslt1-dev \
-    libffi-dev libssl-dev
+COPY . /tmp/server
+RUN cd /tmp/server/ && pip install .
 
-# Enable wsgi module
-RUN a2enmod wsgi
+RUN mkdir /data
+WORKDIR /data
+RUN mkdir -p server/templates \
```

Comment:
This (line23, 24) should not be needed, it was a historical problem that was recently clarified and deemed as unnecessary. This comes from us tho, sorry about the confusion.

Reply:
removed!